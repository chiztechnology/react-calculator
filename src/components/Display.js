import React from 'react';
import './ButtonPanel.css';
import './Calculator.css';

const Display = () => (
        <div className='display-screen'>
            <h3 className='screen-text'>{this.props.value}</h3>
        </div>
    )

export default Display;
