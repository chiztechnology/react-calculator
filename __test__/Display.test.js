import React from 'react';
import renderer from 'react-test-renderer';

import Display from '../src/components/Display';

it('renders correctly when there are no items', () => {
  const tree = renderer
    .create(<Display total="" next="" operation="" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
