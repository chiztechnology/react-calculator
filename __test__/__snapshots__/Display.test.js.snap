// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="display-screen"
>
  <h3
    className="screen-text"
  />
</div>
`;
